packages/manager/apps/web/client/app/hosting/advices/hosting-pro/advice-hosting-pro.controller.js

```diff
@@ -0,0 +1,148 @@
+    this.$translate = $translate;
+    this.Hosting = Hosting;
+    this.OvhApiMe = OvhApiMe;
+    this.RedirectionService = RedirectionService;
+
+    this.CORE_URLS = CORE_URLS;
+  }
+
+  $onInit() {
+    this.advices = this.loadAdvices();
+  }
+
+  loadAdvices() {
+    return this.OvhApiMe.v6()
+      .get()
+      .$promise.then((user) =>
+        this.$q
+          .all([
+            this.getEmailPro(user),
+            this.getWebPerformance(),
+            this.getECommerceDomains(user),
+            this.getSms(),
+            this.getDnsAnycast(),
+            this.getCdn(),
+          ])
+          .then((advices) => compact(advices)),
+      );
+  }
+
+  getEmailPro(user) {
+    return this.$http.get('/email/pro').then(
+      (response) =>
+        !response.data.length
+          ? {
+              localizedName: this.$translate.instant(
+                'hosting_pro_advices_email_pro',
+              ),
+              href: get(
+                this.CORE_URLS,
+                `website_order.email_pro.${user.ovhSubsidiary}`,
+              ),
+              tag: 'cross_sell::hosting::clients_with_prowebhosting::email_pro',
+              external: true,
+            }
+          : null,
+      () => null,
+    );
+  }
+
+  getWebPerformance() {
+    return {
+      localizedName: this.$translate.instant(
+        'hosting_pro_advices_web_performance',
+      ),
+      href: this.$state.href('app.hosting.upgrade', {
+        productId: this.hosting.serviceName,
+      }),
+      tag:
+        'cross_sell::hosting::clients_with_prowebhosting::web_hosting_performance',
+    };
+  }
+
+  getECommerceDomains(user) {
+    return this.Hosting.getTabDomains(this.hosting.serviceName).then(
+      (data) => {
+        const eCommerceDomains = filter(get(data, 'list.results'), (domain) => {
+          return (
+            endsWith(domain.domain, 'shop') || endsWith(domain.domain, 'store')
+          );
+        });
+        const [defaultDomain] = get(data, 'list.results');
+        const domainParts = defaultDomain.domain.split('.');
+        domainParts.splice(-1, 1, 'shop');
+        const shopDomain = domainParts.join('.');
+        return !eCommerceDomains.length
+          ? {
+              localizedName: this.$translate.instant(
+                'hosting_pro_advices_domains',
+              ),
+              href: `${get(
+                this.CORE_URLS,
+                `website_order.domain_search.${user.ovhSubsidiary}`,
+              )}${shopDomain}`,
+              tag:
+                'cross_sell::hosting::clients_with_prowebhosting::domain_store_shop',
+              external: true,
+            }
+          : null;
+      },
+      () => null,
+    );
+  }
+
+  getSms() {
+    return this.$http.get('/sms').then(
+      (response) =>
+        !response.data.length
+          ? {
+              localizedName: this.$translate.instant('hosting_pro_advices_sms'),
+              href: this.RedirectionService.getURL('smsOrder'),
```

Comment:
Import the URL builder
```js
import { buildURL } from '@ovh-ux/ufrontend/url-builder';
```

Build the URL
```js
buildURL('telecom', '#/sms/order');
```

Reply:
This is no longer needed, since the urls have been moved to the 2APIs